extensions/spring-xd-extension-kafka/src/main/java/org/springframework/integration/x/kafka/KafkaPartitionAllocator.java

```diff
@@ -132,6 +132,7 @@ public void afterPropertiesSet() throws Exception {
 		if (STARTED.equals(this.client.getState())) {
 			EnsurePath ensurePath = new EnsurePath(partitionDataPath);
 			ensurePath.ensure(client.getZookeeperClient());
+			// ignore sonar's complaint about this not being synchronized
```

Comment:
No; it's just so the next person that looks at resolving the "issue" will see that it's already been looked at.

I think the rule is too important to disable, but it raises these false positives.